public/src/js/models/collections/article.js

```diff
@@ -378,10 +377,6 @@ define([
                 }, {})
                 .value();
 
-            if (this.group && this.group.parentType === 'Collection') {
-                cleanMeta.group = this.group.index + '';
```

Comment:
this probably has the nice side effect of stop sending group to the server, which is fine.
Can you have a look at unit tests and fix the expectations please?